### ISSUE TITLE
GELF appender modifies the logging data (removes GELF field)

### ISSUE THREAD

**Issue by lafar6502** [log4js-node/log4js-node#330]
When you want to add custom fields to a GELF message you cannot re-use the first argument of the log call because Gelf appender removes the GELF field from it. In the following example only the first log call will behave correctly and add uuid field to the GELF message, further calls will just serialize the first argument and treat it as a message.

``` javascript
var lopts = {GELF: true,  _uuid: '737382'};

log.info(lopts, '1 a message!');
console.log('after send: ', lopts);
log.error(lopts, '1 AN ERROR MESSAGE!');
console.log('after send: ', lopts);
log.error(lopts, '1AN ERROR MESSAGE WITH DATA', {ala: 'ma kota', something: 3989, test: 'a to test'});
log.info(lopts, '1 sent initiate call', 'aaa',' RESULT:', arguments);
```

**Comment by lafar6502** [comment]
but maybe this is by design - remove the GELF field so all other appenders will not log it?

**Comment by nomiddlename** [comment]
I think this issue was fixed in log4js-node/gelf @ 1.0.2
